code/_helpers/unsorted.dm

```diff
@@ -678,7 +678,7 @@ proc/GaussRandRound(var/sigma,var/roundto)
 	if (progbar)
 		qdel(progbar)
 
-/proc/do_after(mob/user as mob, delay as num, needhand = TRUE, atom/movable/act_target = null, use_user_turf = FALSE, display_progress = TRUE, datum/callback/extra_checks)
```

Comment:
Delay as num is actually a proper usage of this, though.

Reply:
No it isn't. `as X` syntax only applies to verbs and procs-as-verbs.